app/src/main/java/org/p2p/wallet/send/model/send_service/SendFeePayerMode.kt

```diff
@@ -0,0 +1,13 @@
+package org.p2p.wallet.send.model.send_service
+
+import com.google.gson.annotations.SerializedName
+
+enum class SendFeePayerMode {
+    @SerializedName("Service")
+    Service,
+    @SerializedName("UserSol")
+    UserSol,
+    @SerializedName("UserSameToken")
+    UserSameToken,
+    Custom
```

Comment:
@SerializedName like for NetworkFeeSource::User?

Reply:
Nope, it's internal enum which shouldn't be serialized or sent to the request